.ci/run_docker.sh

```diff
@@ -1,13 +0,0 @@
-#!/usr/bin/env bash
-
-.ci/before_install_linux.sh
-
-if [[ $TRAVIS_BRANCH == "master" ]]; then
-  DOCKER_BUILDKIT=1 docker build --progress plain -t $DOCKERHUB_REPO:$COMMIT -t $DOCKERHUB_REPO:latest . 
-else
-  DOCKER_BUILDKIT=1 docker build --progress plain -t $DOCKERHUB_REPO:$COMMIT . 
-fi
-
-docker run --rm -v "$PWD:$AVALANCHE_HOME" $DOCKERHUB_REPO:$COMMIT bash "$AVALANCHE_HOME/scripts/build_test.sh"
-docker run --rm -v "$PWD:$AVALANCHE_HOME" $DOCKERHUB_REPO:$COMMIT bash "$AVALANCHE_HOME/scripts/build.sh"
```

Comment:
What I do not know is if building from source and running tests inside the `go` native environment is equivalent than running them in docker.

This was the whole reason for duplication.

If we can be sure that if (linux) build and tests succeed this will also mean that docker build and test will succeed then we can obviously remove this.

Reply:
Docker runs in a linux environment, so there shouldn't be a time that they will succeed in one and not the other. That's not to say that there's no chance whatsoever, but I don't think it justifies running the tests twice. It might be better to run only in docker as opposed to only in linux though. What do you think?

Reply:
If our code uses some platform dependent stuff (e.g. `glibc` for `GCO` or something) then actually it *could* happen that the linux build succeeds (it is based on ubuntu 18.04 I think) while the docker build fails (which is usually on a stripped-down linux version, .e.g. `alpine`). 

However this is the real reason for this PR in the first place, which is to reduce build/CI time. I think we can currently be more efficient and indeed have only one build, but we should be aware of this. Maybe it can be part of some kurtosis test or something in the future. Or maybe we could think of some more extensive test suite before a release which runs more different setups.

But for now let's have just one.